doc/sphinx/user_manual/installation/installing_prerequisites.md

```diff
@@ -71,7 +71,7 @@ There are three main ways to install on Windows.
 The recommended way is to use Linux subsystems for Windows (see <https://docs.microsoft.com/en-us/windows/wsl/install-win10>).
 In this case, start Linux in the Windows terminal and follow the Linux installation description.
 
-If you want to have a native installation, the two main options are using MinGW or Visual Studio.
+If you want to have a native installation, the two main options are using Visual Studio and MinGW. Note that the use of MinGW to compile the world builder is currently not supported (see https://github.com/GeodynamicWorldBuilder/WorldBuilder/issues/743 for more info).
```

Comment:
I would remove mingw support, honestly. You have WSL and VS as two options on windows.

Reply:
yes, that might be better. I don't think it is worth spending more time on fixing this in the future.

Reply:
Thanks, those are good arguments. I have now just removed MinGW support for the world builder.